src/alda/Util.java

```diff
@@ -77,6 +77,26 @@ public static String promptWithChoices(ConsoleReader rdr,
     return m.get(c);
   }
 
+  /**
+   * Small wrapper to make prompts easier to output.
+   * WARNING: This exits the whole program if any read error is given.
+   * @param r The console reader to input on
+   * @param choices A varargs of choices to give
+   * @param prompt The prompt to display above the choice.
+   * @return The choice selected by the user.
+   */
+  public static String uncheckedPromptWithChoices(ConsoleReader r, String prompt, String... choices) {
```

Comment:
I wonder if it would be better to make this an overload of `promptWithChoices`? I would be OK with moving the "unchecked" part (catching IOException and exiting) into the other `promptWithChoices` that this version is calling.